web/client/components/mapcontrols/scale/ScaleBox.jsx

```diff
@@ -25,7 +25,7 @@ var ScaleBox = React.createClass({
     getDefaultProps() {
         return {
             id: 'mapstore-scalebox',
-            scales: mapUtils.getGoogleMercatorScales(0, 21),
+            scales: mapUtils.getGoogleMercatorScales(0, 28),
```

Comment:
The property would need to influence

    scales: mapUtils.getGoogleMercatorScales(0, 28),

in `getDefaultProps` which I don't think is possible? I think that users wanting to customize this should rather simply pass a custom value for `scales` directly, i.e. `mapUtils.getGoogleMercatorScales(0, 21)`.